tgstation.dme

```diff
@@ -600,40 +600,38 @@
 #include "code\FulpstationCode\fulp_world.dm"
 #include "code\FulpstationCode\t5_parts.dm"
 #include "code\FulpstationCode\wintercoats.dm"
-#include "code\FulpstationCode\halloween_costumes\centaur.dm"
```

Comment:
Your DMI's out of date bro

load the DME with DM and it'll automatically fix most of it, then use github's compare to manually tick/untick any remaining differences

Reply:
It looks like they're all still in the .dme they were just organized alphabetically

Reply:
That seems to me like it's gonna cause conflict issues.

Can you manually roll it back and then re-tick the relevant files?

You could just copy your master .dme in here if that works

Reply:
Yeah, I'll try to mess around with it